### PR TITLE
Explain how to launch the bot using Docker

### PR DESCRIPTION
**Pull request by rishi-kai21**
I made an initial commit to the docker issue. please review it and let me know about further changes that can be made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +52,21 @@ After you successfully set up the project, you will have to go through the follo
    * use it as the return value of the method `getBotToken()` inside `StickerifyBot`
 3. Run the `Main` class to start the bot, it will be now able to answer messages in Telegram
 
+
+## Launch The Bot Using Docker
+Prerequisites
+
+Make sure you have already installed both Docker Engine and Docker Compose. You don’t need to install other programs like Python  as they are provided by Docker images.
```
Comment (rob93c) [inline README.md:59]: I would remove the "prerequisites", we can rephrase this line as follows if you like it:
```suggestion
1. Install [Docker](https://docs.docker.com/get-docker/).
```

File: README.md [hunk 2]
```diff
@@ -51,6 +52,21 @@ After you successfully set up the project, you will have to go through the follo
...
+
+1. You can see how to start with docker compose here.
+2. To run the bot locally  with Docker, we need to set Passing Environment Variables Into the Dockerfile you created in 1.
+3. To do that, When we want to set this argument, we pass it with the –build-arg flag:
```
Comment (rob93c) [inline README.md:63]: We can remove point 1 and 2, keeping this as the second one:
```suggestion
2. Run Docker passing it the argument variable with the following command:
```

File: README.md [hunk 3]
```diff
@@ -51,6 +52,21 @@ After you successfully set up the project, you will have to go through the follo
...
+
+docker build -t build-name --build-arg name=arg-name.
+4. Read about it in detail here
```
Comment (rob93c) [inline README.md:66]: There are no details to be read here :trollface: we can remove this point

File: README.md [hunk 4]
```diff
@@ -51,6 +52,21 @@ After you successfully set up the project, you will have to go through the follo
...
+## Launch The Bot Using Docker
+Prerequisites
+
+Make sure you have already installed both Docker Engine and Docker Compose. You don’t need to install other programs like Python  as they are provided by Docker images.
+
+1. You can see how to start with docker compose here.
+2. To run the bot locally  with Docker, we need to set Passing Environment Variables Into the Dockerfile you created in 1.
+3. To do that, When we want to set this argument, we pass it with the –build-arg flag:
+
+docker build -t build-name --build-arg name=arg-name.
```
Comment (rob93c) [inline README.md:65]: To improve its readability I suggest to put this command inside a markdown text with shell syntax, if you are not familiar with them they are written as follows:
````markdown
   ```shell
   docker build -t build-name --build-arg name=arg-name .
   ```
````

Maybe we can also try to be more precise with our command, for instance we can specify that in our case the actual command is 
````shell
docker build -t stickerify --build-arg STICKERIFY_TOKEN={{TOKEN}} .
````
where `{{TOKEN}}` is the token retrieved from [BotFather](https://t.me/BotFather)

**Review by rob93c — CHANGES_REQUESTED** [review]

**Comment by rishi-kai21** [comment]
thank you for giving me better improvement insights. I will make the changes and let you know

**Comment by rishi-kai21** [comment]
also can you please tell me that should I make one pull request for all the changes or should I do it for each separately?

File: README.md (hunk 3 above)
Comment (rishi-kai21) [reply README.md:66]: yeah, i thought I had inserted a hyperlink but it didn't appear, unfortunately :(

**Comment by rob93c** [comment]
> also can you please tell me that should I make one pull request for all the changes or should I do it for each separately?

Please make all the changes in this pull request 😃

**Review by rishi-kai21 — COMMENTED** [review]
All the changes that were asked have been implemented. I would humbly request you to please review them at your leisure. :)

File: src/main/java/com/cellar/stickerify/telegram/Answer.java [hunk 5]
```diff
@@ -14,9 +14,9 @@ public enum Answer {
 			\\- an informative message for any message without a file
 			"""),
 	FILE_READY("""
-            Your sticker file is ready\\!
```
[the remark this answers is not in the document]
Comment (rishi-kai21) [reply src/main/java/com/cellar/stickerify/telegram/Answer.java:17]: No worries :)

File: README.md [hunk 6]
```diff
@@ -80,3 +96,5 @@ See the [**LICENSE**](LICENSE) file for license rights and limitations (MIT).
 <div align="right">
 <sup>made with 💖 by <a href="https://github.com/rob93c">Roberto Cella</a></sup>
 </div>
+
+
```
[the remark this answers is not in the document]
Comment (rishi-kai21) [reply README.md]: I'm sorry for the error. Extra spaces have been resolved

File: README.md (hunk 1 above)
Comment (rishi-kai21) [reply README.md]: prerequisites have been removed and the changes have been made

File: README.md (hunk 2 above)
Comment (rishi-kai21) [reply README.md]: thank you for your suggestion, I have applied your suggestion.

File: README.md (hunk 4 above)
Comment (rishi-kai21) [reply README.md]: Readability has been improved upon

File: README.md [hunk 7]
```diff
@@ -12,6 +12,7 @@ Telegram bot to convert images in the format required to be used as Telegram sti
 * [Hacktoberfest](#Hacktoberfest)
 * [Useful resources](#Useful-resources)
 * [License](#License)
+* [Launch The Bot Using Docker](#How-To-launch-The-Bot-using-Docker)
```
[the remark this answers is not in the document]
Comment (rishi-kai21) [reply README.md]: Reordering has been done

File: src/main/java/com/cellar/stickerify/telegram/Answer.java [hunk 8]
```diff
@@ -26,7 +26,8 @@ public enum Answer {
 			The file conversion was unsuccessful, only _valid image formats_ are supported \\(currently `\\.gif`, `\\.gifv` and `\\.webp` files are not supported\\)\\.
 
 			If you think it should have worked, please report the issue on [Github](https://github.com/rob93c/Stickerify/issues/new/choose)\\.
-			""", true);
```
[the remark this answers is not in the document]
Comment (rishi-kai21) [reply src/main/java/com/cellar/stickerify/telegram/Answer.java:29]: I apologize for my error, the changes have been reverted back.

File: README.md [hunk 9]
```diff
@@ -51,6 +53,19 @@ After you successfully set up the project, you will have to go through the follo
...
+1. Install [Docker](https://docs.docker.com/get-docker/).
+2. Run Docker,passing it the argument variable with the following command:
+   ```shell
+   docker build -t build-name --build-arg name=arg-name 
+   ```
+3. In our case, the specific command will be:
+   ```shell
+   docker build -t stickerify --build-arg STICKERIFY_TOKEN={{TOKEN}} 
+   docker run stickerify
+   ``` where {{TOKEN}} is the token which will be retrieved from BotFather.
```
Comment (rob93c) [inline README.md:67]: I would put the "where" etc on a new line. Also, BotFather should be linked: `[BotFather](https://t.me/BotFather)`

File: README.md [hunk 10]
```diff
@@ -51,6 +53,19 @@ After you successfully set up the project, you will have to go through the follo
    * use it as the return value of the method `getBotToken()` inside `StickerifyBot`
 3. Run the `Main` class to start the bot, it will be now able to answer messages in Telegram
 
+## Launch The Bot Using Docker
+
+1. Install [Docker](https://docs.docker.com/get-docker/).
+2. Run Docker,passing it the argument variable with the following command:
```
Comment (rob93c) [inline README.md:59]:
```suggestion
2. Run Docker, passing it the argument variable with the following command:
```

File: README.md [hunk 11]
```diff
@@ -8,11 +8,13 @@ Telegram bot to convert images in the format required to be used as Telegram sti
 * [How to create a new sticker](#How-to-create-a-new-sticker)
 * [How to set up the project](#How-to-set-up-the-project)
 * [How to run the bot locally](#How-to-run-the-bot-locally)
+* [Launch The Bot Using Docker](#How-To-launch-The-Bot-using-Docker)
 * [How to contribute to the project](#How-to-contribute-to-the-project)
 * [Hacktoberfest](#Hacktoberfest)
 * [Useful resources](#Useful-resources)
 * [License](#License)
 
+
```
Comment (rob93c) [inline README.md:17]: Please remove this empty line

File: README.md [hunk 12]
```diff
@@ -8,11 +8,13 @@ Telegram bot to convert images in the format required to be used as Telegram sti
 * [How to create a new sticker](#How-to-create-a-new-sticker)
 * [How to set up the project](#How-to-set-up-the-project)
 * [How to run the bot locally](#How-to-run-the-bot-locally)
+* [Launch The Bot Using Docker](#How-To-launch-The-Bot-using-Docker)
```
Comment (rob93c) [inline README.md:11]:
```suggestion
* [Launch the bot using Docker](#Launch-the-bot-using-Docker)
```

File: README.md [hunk 13]
```diff
@@ -51,6 +53,19 @@ After you successfully set up the project, you will have to go through the follo
    * use it as the return value of the method `getBotToken()` inside `StickerifyBot`
 3. Run the `Main` class to start the bot, it will be now able to answer messages in Telegram
 
+## Launch The Bot Using Docker
```
Comment (rob93c) [inline README.md:56]:
```suggestion
## Launch the bot using Docker
```

**Review by rob93c — CHANGES_REQUESTED** [review]
The pull request is almost ready, @rishi-kai21 
Don't be impatient, I will approve it as soon as it meets the projects' standards :)

**Comment by rishi-kai21** [comment]
> The pull request is almost ready, @rishi-kai21 Don't be impatient, I will approve it as soon as it meets the projects' standards :)

I am working on the improvements. As soon as I finish them I will let you know. :)

**Review by rishi-kai21 — COMMENTED** [review]
All the changes have been reviewed and implemented. Please review them whenever you have some time off. :)

File: README.md [hunk 14]
```diff
@@ -8,6 +8,7 @@ Telegram bot to convert images in the format required to be used as Telegram sti
 * [How to create a new sticker](#How-to-create-a-new-sticker)
 * [How to set up the project](#How-to-set-up-the-project)
 * [How to run the bot locally](#How-to-run-the-bot-locally)
+* [Launch the bot Using Docker](#Launch-the-bot-using-Docker)
```
Comment (rob93c) [inline README.md:11]:
```suggestion
* [Launch the bot using Docker](#Launch-the-bot-using-Docker)
```

File: README.md [hunk 15]
```diff
@@ -51,6 +52,20 @@ After you successfully set up the project, you will have to go through the follo
...
+2. Run Docker, passing it the argument variable with the following command:
+   ```shell
+   docker build -t build-name --build-arg name=arg-name 
+   ```
+3. In our case, the specific command will be:
+   ```shell
+   docker build -t stickerify --build-arg STICKERIFY_TOKEN={{TOKEN}} 
+   docker run stickerify
+   ```
+   where {{TOKEN}} is the token which will be retrieved from [BotFather](https://t.me/BotFather)
```
Comment (rob93c) [inline README.md:67]:
```suggestion
   where `{{TOKEN}}` is the token which will be retrieved from [BotFather](https://t.me/BotFather)
```

**Review by rob93c — CHANGES_REQUESTED** [review]
There are just a couple of quick changes to do and I will merge this pull request 💎

**Comment by rishi-kai21** [comment]
> There are just a couple of quick changes to do and I will merge this pull request 💎

I apologize for leaving room for errors. I hope now I've made the script error-free. ;)

**Review by rob93c — APPROVED** [review]
The pull request is now perfect, thank you @rishi-kai21! 🎆

**Comment by rishi-kai21** [comment]
> The pull request is now perfect, thank you @rishi-kai21! 🎆

thank you for letting me contribute to your beautiful project. Much appreciated :)

**Pull request merged**